gr-fft/python/fft/logpwrfft.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, sample_rate, fft_size, ref_scale, frame_rate, avg_alpha, aver
             avg_alpha: FFT averaging (over time) constant [0.0-1.0]
             average: Whether to average [True, False]
             win: the window taps generation function
+            shift: enable re-ordering of FFT output to "natural" order
```

Comment:
That's a confusing comment. What is the "natural" order? I would suggest that the comment be "shift: enable re-ordering of FFT output to place the zero-frequency component at the center of the spectrum"